napalm/nxos/nxos.py

```diff
@@ -400,6 +400,12 @@ def get_interfaces(self):
                 is_up = interface_details.get('admin_state', '') == 'up'
             else:
                 is_up = interface_details.get('state', '') == 'up'
+            if 'eth_mtu' in interface_details:
+                mtu = int(interface_details['eth_mtu'])
+            elif 'svi_mtu' in interface_details:
```

Comment:
That is a bit awkward (but I understand why, I know the NX-API slickness). Would you please add a comment around that with more details, and - if applicable - what versions have one but not the other etc.